src/interfaces/IERC7246.sol

```diff
@@ -12,7 +12,7 @@ interface IERC7246 {
     event Encumber(address indexed owner, address indexed taker, uint256 amount);
 
     /**
-     * @dev Emitted when the encumbrance of a `taker` to an `owner` is reduced
+     * @dev Emitted when the encumbrance of an `owner` to a `taker` is reduced
```

Comment:
I think this was not meant to be reverted

Reply:
I included this intentionally. That's the correct definition, right? from `owner` to `taker`?

Reply:
o yah u right my bad